src/views/Editor.vue

```diff
@@ -46,99 +47,103 @@ export default {
     return {
       editor: null,
       placeholder: new Doc({ text: formatTags(this.$store.state.context.tags, " ") }),
-    }
+    };
   },
   watch: {
     doc() {
-      this.$refs.editable.clearHistory()
-      this.$refs.editable.focusEditor()
+      this.$refs.editable.clearHistory();
+      this.$refs.editable.focusEditor();
     },
     tags: {
       deep: true,
       handler() {
-        this.updateTitle()
+        this.updateTitle();
       },
     },
     header() {
-      this.updateTitle()
-    }
+      this.updateTitle();
+    },
   },
   computed: {
     appearance() {
-      return this.$store.state.settings.theme === 'october' ? 'dark' : this.$store.state.settings.theme
+      return this.$store.state.settings.theme === "october"
+        ? "dark"
+        : this.$store.state.settings.theme;
+
+      // do I add multiple returns for the auto light and dark within here?
```

Comment:
This method has to return either `'light'` or `'dark'` for the Ink editor to work correctly. You can determine which to do with matchMedia again.

```suggestion
      if (this.$store.state.settings.theme === 'light') { return 'light' }
      if (this.$store.state.settings.theme === 'auto') {
        const isDark = window.matchMedia('(prefers-color-scheme: dark)').matches
        
        return isDark ? 'dark' : 'light'
      }

      // dark should be the default
      return 'dark'
```